docs/blog/uf2-for-arduino-uno.md

```diff
@@ -0,0 +1,124 @@
+![Resetting UNO](/static/blog/fork.jpg)
+
+# Bootloader with a fork
+
+**Posted on October 17, 2017 by [mmoskal](https://github.com/mmoskal)**
+
+After building the [UF2 bootloader for SAMD21](https://makecode.com/blog/one-chip-to-flash-them-all), 
+I had the pleasure of scaling
+it up to a particular Linux embedded board
+and now I was able to scale it down to the AVR level, namely the world-famous
+Arduino UNO. 
+
+The UNO has two AVR chips -- the main one is ATmega328p (the big thing)
+and ATmega16u2 dedicated to handling USB (it was a smaller ATmega8u2 in
+previous iterations of UNO, but is the version with 16k of flash in the most recent
+and common R3).
+
+The main chip sports a 512 byte bootloader (called [optiboot](https://github.com/Optiboot/optiboot))
```

Comment:
supports?

Reply:
sport something-to have or wear something in a proud way